africanus/model/spi/tests/test_component_spi.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `codex-africanus` package."""
+
+import numpy as np
+import pytest
+
+
+def test_vs_scipy():
```

Comment:
Rename to `test_fit_spi_components_vs_scipy`. This is so that there's a fairly unique test name which allows us to run `py.test -s -v -k test_fit_api_components_vs_scipy`